addons/calendar/static/src/js/base_calendar.js

```diff
@@ -122,7 +122,18 @@ BasicModel.include({
      */
     _fetchSpecialAttendeeStatus: function (record, fieldName) {
         var context = record.getContext({fieldName: fieldName});
-        var attendeeIDs = record.data[fieldName] ? this.localData[record.data[fieldName]].res_ids : [];
```

Comment:
could you just write const attendeeIDs = record.data[fieldName] ? record.data[fieldName].res_ids : [];?
here I meant. instead of
        let attendeeIDs = [];
        if (record.data[fieldName]) {
        ...

Reply:
@Polymorphe57 This will not always be true, ```record.data[fieldName]``` will be datapoint ID in case if you open same event in form view(go to listview and click on any record), so in that case we need to fetch record from localData